src/super_gradients/training/utils/collate_fn/ppyoloe_collate_fn.py

```diff
@@ -16,13 +16,19 @@ class PPYoloECollateFN(DetectionCollateFN):
     Collate function for PPYoloE training
     """
 
-    def __init__(self, random_resize_sizes: Union[List[int], None] = None, random_resize_modes: Union[List[int], None] = None):
+    def __init__(
```

Comment:
Maybe its a good opprotunity to make PPYoloECollateFN DetectionCollateFN since it essentially hides our multiscale feature (which is unfortunate).